docs/usermanuals/tr/reference/cli.md

```
@@ -57,94 +56,135 @@ En sık kullanılan pattern'ler:
     [--remove-duplicates] [--remove-cross-split-overlap=val|test] \
     [--output-clean PATH] \
     [--show-leakage] \
-    [--sample-rate FLOAT]
+    [--sample-rate FLOAT] \
+    [--pii-ml] [--pii-ml-language LANG] \
+    [--croissant] \
+    [--verbose]
 ```
 
-Tam semantic için bkz. [Veri Seti Denetimi](#/data/audit).
+`--workers N` split düzeyinde paraleliz; on-disk JSON worker sayısından bağımsız olarak byte-identical (sadece `generated_at` zamanı değişir). Tam semantik için bkz. [Veri Denetimi](#/data/audit).
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)